packages/BetterPhpDocParser/PhpDocParser/ConstExprClassNameDecorator.php

```diff
@@ -30,7 +30,7 @@ public function decorate(PhpDocNode $phpDocNode, PhpNode $phpNode): void
     {
         $this->phpDocNodeTraverser->traverseWithCallable($phpDocNode, '', function (Node $node) use (
             $phpNode
-        ): Node|null {
+        ): ?Node {
```

Comment:
The rule changes should be reverted as well

Reply:
Ok, reverted https://github.com/rectorphp/rector-src/pull/3231/commits/755100ccf04af7b7bd8d47f762e840603d37cd1d https://github.com/rectorphp/rector-src/pull/3231/commits/c74239f480cd1f0cf3f9fc92c4708fd9e9d09f1f